storage/Cargo.toml

```diff
@@ -52,6 +52,8 @@ sha3 = { version = "0.10.8", optional = true }
 criterion = { workspace = true, features = ["async_tokio", "html_reports"] }
 pprof = { workspace = true, features = ["flamegraph"] }
 rand.workspace = true
+rlp = "0.6.1"
+sha3 = "0.10.8"
```

Comment:
I don't see where these are getting used, and I'd much prefer not having rlp or sha3 in storage.